app/controllers/Api.scala

```diff
@@ -223,6 +223,14 @@ class Api (cc: ControllerComponents, prismDataStore: Prism, prismConfiguration:
       Api.singleItem(prismDataStore.vpcAgent, arn)
     }
 
+    def cloudformationStackList = Action.async { implicit request =>
+      Api.itemList(prismDataStore.cloudformationStackAgent, "cloudformation-stacks")
```

Comment:
Spotting whilst testing on CODE - kebab casing here makes the `jq` query a bit annoying as you have to [wrap the field in quotes](https://stackoverflow.com/a/37344498/3868241):

`curl https://prism.../cloudformationStacks | jq '.data."cloudformation-stacks"[].outputs'`

Could this be `cloudformationStacks` instead?

`curl https://prism.../cloudformationStacks | jq '.data.cloudformationStacks[].outputs'`

Not a blocker.